src/network/asyncore_pollchoose.py

```diff
@@ -75,13 +73,16 @@
 except (ImportError, AttributeError):
     WSAECONNRESET = ECONNRESET
 try:
-    from errno import WSAEADDRINUSE
+    # side-effects on Windows or cruft?
+    from errno import WSAEADDRINUSE  # pylint: disable=unused-import
 except (ImportError, AttributeError):
     WSAEADDRINUSE = EADDRINUSE
+import helper_random
```

Comment:
Why it was moved here?

Reply:
Recall this commit was a WIP. All imports now automatically sorted. If you have any import sorting issues from now on you will have to find suitable isort settings. There will be no more manual import sorting.